cou/cli.py

```diff
@@ -61,17 +64,29 @@ def parse_args(args: Any) -> argparse.Namespace:
         dest="loglevel",
         choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
         type=str.upper,
-        help="Set the logging level",
+        help="Set the logging level. Defaults to INFO. This only affects stdout. The logfile "
```

Comment:
What is COU_DATA? Are we mention it somewhere in the help? Can we provide the actual path with f-string?